Tests/CombineWampTests/IntegrationTest.swift

```diff
@@ -0,0 +1,87 @@
+import Combine
+import CombineWamp
+import Foundation
+import XCTest
+
+final class IntegrationTest: XCTestCase {
+    let serialization: WampSerializing = {
+        let decoder = JSONDecoder()
+        let encoder = JSONEncoder()
+        return WampSerializing.json(decoder: { decoder }, encoder: { encoder })
+    }()
+
+    func transport() -> WampTransport {
+        let router = URL(string: "ws://localhost:8080/ws")!
```

Comment:
Real networking in a unit test?

Reply:
Integration Test.

It could (should) be a different target, but for now I wanted to make it work first and ensure that it's compatible with Crossbar and wampcc. Both, Crossbar and wampcc violate the protocol here and there, so it's important to test against a real thing.

Soon I hope I'm gonna implement the Router roles and then the integration tests can be completely automated.